index.html

```diff
@@ -17,6 +17,12 @@
                 <label class="hamberger-btn" >
                 <img class="hamberger" id="ham" src="img/IconMenu.png">
                 </label>
+                <ul class="nav-links">
+                  <li><a href="#"></a>Portfolio</li>
+                  <li><a href="#"></a>About</li>
+                  <li><a href="#"></a>Contact</li>
+                  <a><img src="img/Union.png"><a>
+                </ul>
```

Comment:
- [ ] Kindly resolve the interference of the destop nav-items in the mobile view, I believe you suppose to hide them. ℹ️ 

<img width="277" alt="Untitled" src="https://github.com/Obimbo07/Portfolio-2/assets/103386183/065f6603-be22-4e88-b53b-c5d4f0e8c364">